Conference-Android/app/src/main/AndroidManifest.xml

```diff
@@ -47,12 +47,12 @@
         <activity
```

Comment:
OK, so up goes all the way back to the conference list, while back just goes back one.